eclipse-scout-core/src/form/fields/smartfield/SmartField.js

```diff
@@ -1341,6 +1328,17 @@ export default class SmartField extends ValueField {
     this.setProperty('activeFilter', this.activeFilterEnabled ? activeFilter : null);
   }
 
+  _renderActiveFilter() {
```

Comment:
Is this a realistic case? If I simulate it (update activeFilter while popup is open), the nodes are updated but the active filter bar will be removed.